server/src/main/java/com/linecorp/centraldogma/server/metadata/UserAndTimestamp.java

```diff
@@ -89,6 +89,13 @@ public String timestamp() {
         return timestampAsText;
     }
 
+    /**
+     * Returns the epoch milliseconds of the {@link #timestamp()}.
+     */
+    public long epochMillis() {
```

Comment:
nit; `timestampMillis` may be a better name since it represents both the field and how it is represented